internal/pkg/mount/v2/pseudo.go

```diff
@@ -21,6 +21,9 @@ func Pseudo() Points {
 		NewPoint("tmpfs", "/run", "tmpfs", WithFlags(unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_RELATIME), WithData("mode=0755")),
 		NewPoint("tmpfs", "/system", "tmpfs", WithData("mode=0755")),
 		NewPoint("tmpfs", "/tmp", "tmpfs", WithFlags(unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_NODEV), WithData("size=64M"), WithData("mode=0755")),
+		// TODO: mount this under /sys/fs/selinux as in most distros?
+		// That'd be harder to move when we switch root
```

Comment:
good question, I wasn't clear if `Move` moves correctly sub-mounts, so if we move `/sys`, would it move `/sys/fs/selinux` as well?

If it does, we can probably simply move selinux mount to a different set of points which we mount, but don't move

Reply:
I will send a PR to fix this